index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title></title>
+    <title>Portfolio</title>
```

Comment:
It would be nice if this had a bit more detail. e.g. 'Tanveer Bassi: Professional Portfolio. ' or along those lines. Make it more about you.